pkgs/applications/networking/mumble/default.nix

```diff
@@ -108,6 +108,17 @@ let
       sha256 = "sha256-SYsGCuj3HeyAQRUecGLaRdJR9Rm7lbaM54spY/zx0jU=";
       fetchSubmodules = true;
     };
+
+    patches = [
+      # fixes 'static assertion failed: static_assert(sizeof(CCameraAngles) == 0x408, "");'
+      # when compiling pkgsi686Linux.mumble, which is a dependency of x64 mumble_overlay
+      # https://github.com/mumble-voip/mumble/pull/5850
+      # Remove with next version update
+      (fetchpatch {
+        url = "https://github.com/mumble-voip/mumble/commit/13c051b36b387356815cff5d685bc628b74ba136.patch";
+        hash = "sha256-S97UgvZXNGEJPM9yYbacHhIZEqqXiVf2MroZxWjZRBI=";
+      })
```

Comment:
```
building '/nix/store/5508mgiigpyjvmlf3dqbf33c1aa4b0c6-13c051b36b387356815cff5d685bc628b74ba136.patch.drv'...

trying https://github.com/mumble-voip/mumble/commit/13c051b36b387356815cff5d685bc628b74ba136.patch
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   890  100   890    0     0   3231      0 --:--:-- --:--:-- --:--:--  3236
error: hash mismatch in fixed-output derivation '/nix/store/5508mgiigpyjvmlf3dqbf33c1aa4b0c6-13c051b36b387356815cff5d685bc628b74ba136.patch.drv':
         specified: sha256-S97UgvZXNGEJPM9yYbacHhIZEqqXiVf2MroZxWjZRBI=
            got:    sha256-Rq8fb6NFd4DCNWm6OOMYIP7tBllufmQcB5CSxPU4qqg=
error: 1 dependencies of derivation '/nix/store/w2fplj51sbjqg7gw8kg88fxdwacgf18r-mumble-1.4.287.drv' failed to build
```

:thinking:

Reply:
what the
it works for me, I don't have the slightest idea what's happening 😅 https://gist.github.com/raphaelr/7c230a1c326277cd5360933b4ab0edb2

Reply:
I'm an idiot... I used nix-prefetch-url and forgot that fetchpatch does postprocessing m(